Gemfile

```diff
@@ -43,6 +43,7 @@ end
 
 group :test do
   gem "capybara-webkit", ">= 1.2.0"
+  gem "cucumber-rails", :require => false # Cucmber features
```

Comment:
I think we should install `capybara` as well in this stage as it would be required later for writing cucumber steps